include/quicr/quicr_client.h

```diff
@@ -232,10 +218,11 @@ class QuicRClient
    * @param auth_token            : Auth Token to valiadate the Subscribe Request
    * @param payload               : Opaque payload to be forwarded to the Origin 
    */
-  bool publish_intent(const QUICRNamespace& quicr_namespace,
-                      const std::string& origin_url,      
+  bool publishIntent(const QUICRNamespace& quicr_namespace,
+                      const std::string& origin_url,
```

Comment:
I realized that actually, for relays with no configuration, like one in my hope AP, it would need to  know the origin_url. Most relays that actually authorized things would ignore this.